### PR TITLE
monikadesign: add new private tracker

### PR DESCRIPTION
**Pull request by xiaoxi654**

### CODE REVIEW

File: src/Jackett.Common/Definitions/monikadesign-api.yml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        75%: 0.25
+        100%: 0 # freeleech
+        "*": 0 # catch errors
+    uploadvolumefactor:
+      # api returns 0=false, 1=true
+      selector: double_upload
+      case:
+        0: 1 # normal
+        1: 2 # double
+# json UNIT3D 6.0.4
```
[the remark this answers is not in the document]
Comment (garfield69) [reply src/Jackett.Common/Definitions/monikadesign-api.yml:158]: What are the seeding time requirements for downloaded torrents, and/or is there a minimum share ratio?
for example, MST 48 hours, MR 1.0

**Review by garfield69 — COMMENTED** [review]
If the site is really a unit3d 6.4.1 as suggested by the login page eye catcher, then 
we should be able to use `category: selector: category_id` instead of `categorydesc: selector: category`
and `poster: selector: meta.poster` instead of `poster: selector: poster`

unless the site has a 6.4.1 app.js and is still using a 6.0.4 unit3d.js, which is an odd combo but not out of the realms of possibility.

Could you please provide a HTML dump of https://monikadesign.uk/torrents?perPage=25 so we can do some cross checking?
See https://github.com/Jackett/Jackett/wiki/Troubleshooting/#how-to-provide-html-source 
Thanks.

File: src/Jackett.Common/Definitions/monikadesign-api.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (xiaoxi654) [reply src/Jackett.Common/Definitions/monikadesign-api.yml:158]: > What are the seeding time requirements for downloaded torrents, and/or is there a minimum share ratio?
> for example, MST 48 hours, MR 1.0

There’s no limit about seeding time or single seed share ratio requirement.

**Comment by xiaoxi654** [comment]
> If the site is really a unit3d 6.4.1 as suggested by the login page eye catcher, then we should be able to use `category: selector: category_id` instead of `categorydesc: selector: category` and `poster: selector: meta.poster` instead of `poster: selector: poster`
> 
> unless the site has a 6.4.1 app.js and is still using a 6.0.4 unit3d.js, which is an odd combo but not out of the realms of possibility.
> 
> Could you please provide a HTML dump of https://monikadesign.uk/torrents?perPage=25 so we can do some cross checking? See https://github.com/Jackett/Jackett/wiki/Troubleshooting/#how-to-provide-html-source Thanks.

It’s more like I forget to change the version in comment, I have only copy-pasted an another unit3d api definition. I’ll check those suggestions when I have time, thanks!

File: src/Jackett.Common/Definitions/monikadesign-api.yml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+  rows:
+    selector: data
+    attribute: attributes
+    count:
+      selector: meta.total
+
+  fields:
+    categorydesc:
+      selector: category
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/monikadesign-api.yml:105]:
```suggestion
      selector: category_id
```

File: src/Jackett.Common/Definitions/monikadesign-api.yml [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    title:
+      selector: name
+    details:
+      selector: details_link
+    download:
+      selector: download_link
+    poster:
+      selector: poster
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/monikadesign-api.yml:113]:
```suggestion
      selector: meta.poster
```

**Review by garfield69 — CHANGES_REQUESTED** [review]

File: src/Jackett.Common/Definitions/monikadesign-api.yml [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+      args: ["\\.", " "]
+
+  rows:
+    selector: data
+    attribute: attributes
+    count:
+      selector: meta.total
+
+  fields:
+    categorydesc:
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/monikadesign-api.yml:104]:
```suggestion
    category:
```

**Review by garfield69 — CHANGES_REQUESTED** [review]

**Review by garfield69 — APPROVED** [review]

**Pull request merged**